diff --git a/src/bindings.rs b/src/bindings.rs
index 1533805..3bdd6df 100644
--- a/src/bindings.rs
+++ b/src/bindings.rs
@@ -1,3 +1,11 @@
+#![allow(
+    clippy::all,
+    dead_code,
+    unused_variables,
+    non_snake_case,
+    unused_braces,
+    unused_mut
+)]
 pub mod r#root {
     pub mod r#component {
         pub trait RootExports<I: RootImports + ::core::marker::Send> {
@@ -1148,6 +1156,7 @@ impl<
             >>::T,
         >,
     ) -> () {
+        let mut rts = self.rt.lock().unwrap();
         let ret = ::hyperlight_host::sandbox::Callable::call::<
             ::std::vec::Vec<u8>,
         >(
